src/families/ethereum/modules/send.js

```diff
@@ -68,6 +69,10 @@ const send: ModeModule = {
           result.errors.amount = new NotEnoughBalance();
         }
       }
+
+      if (!result.errors.amount && !t.feesStrategy) {
+        result.errors.feesStrategy = new NoFeeStrategySelected();
```

Comment:
i'm not sure to understand the need of this this error?
to me this error is not possible if we settle feesStategy to a default, which we should as discussed earlier.

Reply:
Well I add to cli-transaction a default value that would be set only on CLI, and stay undefined in LLM / LLD.
It was asked to force the user to select between one of these strategies in LLM and LLD.
Maybe should I just import that logic in both LLM / LLD ?

Reply:
the way to solve this is to put it as part of the "initTransaction" and not in the CLI inferTransactions

Reply:
createTransaction i mean https://github.com/LedgerHQ/ledger-live-common/blob/7ad2f9be55118ce258ded3f58bf85447be1da5d2/src/families/bitcoin/bridge/libcore.js#L62

Reply:
Ok spec have change, so now we set "medium" by default and user will have "medium" by default, so we can remove the error 👍